packages/python-cryptography/build.sh

```diff
@@ -3,14 +3,15 @@ TERMUX_PKG_DESCRIPTION="Provides cryptographic recipes and primitives to Python
 TERMUX_PKG_LICENSE="Apache-2.0, BSD 3-Clause"
 TERMUX_PKG_LICENSE_FILE="LICENSE, LICENSE.APACHE, LICENSE.BSD"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="42.0.8"
+TERMUX_PKG_VERSION="43.0.0"
 TERMUX_PKG_SRCURL=https://github.com/pyca/cryptography/archive/refs/tags/${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=38ee4ce0804e4003e3093db8342cd7e6ee65614c8bbf70c98f1716e0f33709ed
+TERMUX_PKG_SHA256=e24209db3db62446ce517bb2bfac2709b3d952c5e4601aa653337de9cc666dc7
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS="openssl, python, python-pip"
 TERMUX_PKG_BUILD_IN_SRC=true
 TERMUX_PKG_UPDATE_TAG_TYPE="newest-tag"
-TERMUX_PKG_PYTHON_COMMON_DEPS="wheel, cffi, setuptools-rust"
+TERMUX_PKG_BUILD_DEPENDS="python-maturin"
```

Comment:
No idea why this has no effect… pip still installs maturin in case `python-maturin` installed

Reply:
This PR is not merged yet...

Reply:
This will not work as you expected. To build python-cryptography, a x86_64-linux-gnu maturin is needed to run on the host (and it is installed because of TERMUX_PKG_PYTHON_COMMON_DEPS), but python-maturin provides the one targeting aarch64-linux-android.